README.md

```diff
@@ -1,73 +1,19 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+My Portfolio
 
```

Comment:
- [ ] [REQUIRED]. As suggested by the previous code reviewer. I also kindly suggest that you use [markdown](https://www.markdownguide.org/) to write your readme file and use the Microverse Template. You can copy the raw files directly from the template, then edit them as seen in this demo video:

https://user-images.githubusercontent.com/52934128/163938226-d191d639-9636-46f4-bb46-762b8871ab45.mp4



Here is an example of how you would write your README file:

```
![](https://img.shields.io/badge/Microverse-blueviolet)

# Portfolio

> Portfolio set up: mobile version Skeleton.

Additional description about the project and its features.

## Built With

- HTML, CSS
- Stylelint, Web Hint, Lighthouse


## Getting Started
To get a local copy up and running, follow the steps in Setup below.

### Prerequisites
Basic knowledge of HTML, CSS, and JavaScript.

### Setup
If you installed git you can clone the code to your machine, or download a ZIP of all the files directly.

[Download the ZIP from this location](https://github.com/Bushido-brown/My-Portfolio2/archive/refs/heads/master.zip), or run the following [git](https://git-scm.com/downloads) command to clone the files to your machine:

```bash
git clone https://github.com/Bushido-brown/My-Portfolio2.git
```

- Once the files are on your machine, open the _Portfolio_ folder in [Visual Studio Code](https://code.visualstudio.com/).

- With the files open in Visual Studio Code, press the **Go Live** button at the bottom of the window to launch the files with [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer).

### Install

The following tools help make easier to work with sample code.

- [git](https://git-scm.com/downloads): A tool for managing source code
- [Visual Studio Code](https://code.visualstudio.com/): A source code editor
- [Live Server](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer): A simple web server utility for Visual Studio Code

## Authors

👤 **Temitope Ogunleye**

- GitHub: [@githubhandle](https://github.com/Bushido-brown)
- Twitter: [@twitterhandle](https://twitter.com/tuzyorizu)
- LinkedIn: [LinkedIn](https://linkedin.com/in/Tochukwu Orizu)


## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](https://github.com/Bushido-brown/My-Portfolio2/issues).

## Show your support

Give a ⭐️ if you like this project!


## 📝 License

This project is [MIT](./MIT.md) licensed.

```